archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -0,0 +1,404 @@
+        -D LLVM_ENABLE_RTTI=ON \
+        -D LLVM_ENABLE_FFI=ON \
+        -D LLVM_BINUTILS_INCDIR=/usr/include \
+        -D FFI_INCLUDE_DIR="$(pkg-config --variable=includedir libffi )" \
+        -D LLVM_BUILD_LLVM_DYLIB=ON \
+        -D LLVM_LINK_LLVM_DYLIB=ON \
+        -D LLVM_INSTALL_UTILS=ON \
+        -D LLVM_BUILD_DOCS=ON \
+        -D LLVM_INCLUDE_DOCS=ON \
+        -D LLVM_ENABLE_DOXYGEN=OFF \
+        -D CLANG_INCLUDE_DOCS=ON \
+        -D LLVM_EXPERIMENTAL_TARGETS_TO_BUILD=AVR \
+        -D LLVM_ENABLE_SPHINX=ON \
+        -D SPHINX_OUTPUT_HTML=OFF \
+        -D SPHINX_WARNINGS_AS_ERRORS=OFF \
+        -D POLLY_ENABLE_GPGPU_CODEGEN=ON \
+        -D LLVM_VERSION_SUFFIX="" \
+        -D LLDB_USE_SYSTEM_SIX=1 \
+        -D LLVM_ENABLE_PROJECTS="lldb;polly;compiler-rt;lld;clang-tools-extra;clang"
+#       -D TENSORFLOW_C_LIB_PATH="/usr/include/tensorflow/"
+
+    ninja -C _build64 LLVMgold  all ocaml_doc
+    DESTDIR="$srcdir/fakeinstall_64" ninja -C _build64 install
+
+    export PKG_CONFIG_PATH="/usr/lb32/pkgconfig"
+
+    LIB32_CFLAGS="$CFLAGS"" -m32"
+    LIB32_CXXFLAGS="$CXXFLAGS"" -m32"
+
+    cmake \
+    -B _build32 \
+    -S llvm-project/llvm \
+    -G Ninja \
+        -D LLVM_ENABLE_PROJECTS="clang;clang-tools-extra;compiler-rt" \
+        -D CMAKE_BUILD_TYPE=Release \
+        -D CMAKE_INSTALL_PREFIX=/usr \
+        -D LLVM_LIBDIR_SUFFIX=32 \
+        -D CMAKE_C_FLAGS="$LIB32_CFLAGS" \
+        -D CMAKE_CXX_FLAGS="$LIB32_CXXFLAGS" \
+        -D LLVM_TARGET_ARCH:STRING=i686 \
+        -D LLVM_HOST_TRIPLE="$CHOST" \
+        -D LLVM_DEFAULT_TARGET_TRIPLE="i686-pc-linux-gnu" \
+        -D LLVM_BINUTILS_INCDIR=/usr/include \
+        -D LLVM_BUILD_LLVM_DYLIB=ON \
+        -D LLVM_LINK_LLVM_DYLIB=ON \
+        -D LLVM_ENABLE_RTTI=ON \
+        -D LLVM_ENABLE_FFI=ON \
+        -D LLVM_BUILD_TESTS=OFF \
+        -D LLVM_BUILD_DOCS=OFF \
+        -D LLVM_ENABLE_SPHINX=OFF \
+        -D LLVM_ENABLE_DOXYGEN=OFF \
+        -D FFI_INCLUDE_DIR="$(pkg-config --variable=includedir libffi)" \
+        -D LLVM_BINUTILS_INCDIR=/usr/include \
+        -D LLVM_APPEND_VC_REV=ON 
+
+    ninja -C _build32 all ocaml_doc LLVMgold
+    DESTDIR="$srcdir/fakeinstall_32" ninja -C _build32 install
+}
+
+_fakeinstall_64 () {
+        local src f dir
+        for src; do
+        f="${src#fakeinstall_64/}"
+        dir="$pkgdir/${f%/*}"
+        install -m755 -d "$dir"
+        mv -v "$src" "$dir/"
+       done
+}
+
+_fakeinstall_32 () {
+         local src f dir
+         for src; do
+         f="${src#fakeinstall_32/}"
+         dir="$pkgdir/${f%/*}"
+         install -m755 -d "$dir"
+         mv -v "$src" "$dir/"
+       done
+}
+
+package_lldb-git() {
+     pkgdesc="Next generation, high-performance debugger (git version)"
+     url="https://lldb.llvm.org/"
+     depends=("clang-git=$pkgver-$pkgrel" "llvm-libs-git=$pkgver-$pkgrel")
+     optdepends=('llvm-git: for the man file')
+     provides=("lldb=$pkgver")
+     conflicts=('lldb')
```

Comment:
Missing replaces and provides for the svn version? (or have they never existed before?)

Same for `lld`, `lib32-clang`, `lib32-llvm` below (I feel like if `lib32-llvm-libs-svn` existed `lib32-llvm-svn` probably did as well...)